test/README.md

```diff
@@ -7,7 +7,7 @@ This directory contains tests and testing docs for `Knative Serving`:
 - [End-to-end tests](#running-end-to-end-tests), of which there are two types:
   - Conformance tests in [`/test/conformance`](./conformance)
   - Other end-to-end tests in [`/test/e2e`](./e2e)
-- [Performance tests](./performance)
+- [Performance tests](#running-performance-tests)
```

Comment:
similar to "end to end tests" section, can you describe the location of performance test here?